packages/components/video/UploadVideoButton.tsx

```diff
@@ -58,6 +58,6 @@ const buttonContainerStyle: ViewStyle = {
 };
 const buttonTextStyle: TextStyle = {
   ...fontSemibold14,
-
+  textTransform: "capitalize",
```

Comment:
I'm not sure about this 
![image](https://github.com/TERITORI/teritori-dapp/assets/50441633/0f978472-400a-41ab-a994-2917ad5ad537)